includes/APIWrite.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+namespace MediaWiki\Extension\Example;
+
+use MediaWiki\Rest\SimpleHandler;
+use Wikimedia\ParamValidator\ParamValidator;
+use MediaWiki\MediaWikiServices;
+
+use Title;
+use WikitextContent;
+use WikiPage;
+
+# include / exclude for debugging
+error_reporting(E_ALL);
+ini_set("display_errors", 1);
+
+require_once("ApiUtil.php");
+require_once("Util.php");
+
+function selectToArray($db, $table, $col, $conds) {
```

Comment:
Is this used in APIWrite and APIRead, or only here?

Reply:
Only in write only. Needs to be removed in read.

Same true for domain manifest.

Reply:
Open. Let's check together!